### ISSUE TITLE
Question regarding val/test split

### ISSUE THREAD

**Issue by joshinh** [thanhan/seqcrowd-acl17#3]
Thanks for releasing the code!

I had a question regarding the val/test split. For task 1 of NER, what exactly is the training data being used to learn the HMM parameters - If I understand correctly the val set is used initially to tune the hyper-parameters of the model. Once that is done, the *test data* is used to learn the HMM parameters using the best hyper parameters found, and then the results are reported using the gold labels available for the test set. Is that correct?

Sorry if the question seems silly, but I couldn't find in the code where the HMM parameters were being saved after training on the val set so got confused.

**Comment by thanhan** [comment]
Yes, that's correct.

**Comment by joshinh** [comment]
I see, thanks for the clarification! Is there any reason why this procedure was followed as compared to the standard process of using the trained HMM parameters obtained from training on val set (and ideally tuned a separate val set)? Because according to the current setup, one will have retrain the complete model every time one needs to use it for aggregation of labels on a new set.
